Improve resiliency of schema overview in postgres (#23664)
directus/directus: .changeset/fifty-bobcats-beg.md

---
'@directus/schema': patch
---

> Fixed an issue that would cause directus to crash on unexpected database schema contents

Reviewer: (I'd maybe mention that it only concerns PostgreSQL databases

```suggestion
Fixed an issue that would cause a server crash on unexpected PostgreSQL database schema contents
```

)
